projects/packages/paypal-payments/src/block/utils.js

```diff
@@ -0,0 +1,62 @@
+import { trimEnd } from 'lodash';
+import { getCurrencyDefaults } from '../../shared/currencies';
+import { SIMPLE_PAYMENTS_PRODUCT_POST_TYPE } from './constants';
+
+export const isValidSimplePaymentsProduct = product =>
+	product.type === SIMPLE_PAYMENTS_PRODUCT_POST_TYPE && product.status === 'publish';
+
+// based on https://stackoverflow.com/a/10454560/59752
+export const decimalPlaces = number => {
+	const match = ( '' + number ).match( /(?:\.(\d+))?(?:[eE]([+-]?\d+))?$/ );
+	if ( ! match ) {
+		return 0;
+	}
+	return Math.max( 0, ( match[ 1 ] ? match[ 1 ].length : 0 ) - ( match[ 2 ] ? +match[ 2 ] : 0 ) );
+};
+
+const getNavigatorLanguage = ( defaultLang = 'en-US' ) => {
+	const nav = global?.window?.navigator;
+	if ( nav?.languages?.length ) {
+		return nav.languages[ 0 ];
+	}
+	return nav?.userLanguage ?? nav?.language ?? nav?.browserLanguage ?? defaultLang;
+};
+
+// Legacy method of displaying prices.
+export const formatPriceFallback = ( price = 0, currency, withSymbol = true ) => {
+	const { precision, symbol } = getCurrencyDefaults( currency );
+	const value = price.toFixed( precision );
+	// Trim the dot at the end of symbol, e.g., 'kr.' becomes 'kr'
+	return withSymbol ? `${ value } ${ trimEnd( symbol, '.' ) }` : value;
+};
+
+// Display prices using Intl.NumberFormat if available - supported in 95.75% of browsers as of Oct 2020.
+export const formatPrice = ( price = 0, currency, withSymbol = true ) => {
+	if ( ! window.Intl || 'function' !== typeof Intl.NumberFormat ) {
+		return formatPriceFallback( price, currency, withSymbol );
+	}
+
+	const siteLocale = window?.Jetpack_Editor_Initial_State?.siteLocale ?? 'en-US';
```

Comment:
It may be worth adding a to-do item here. Eventually you'll need the package to pass its own data from PHP to JS, instead of relying on the existing Jetpack bundle.

Reply:
This is a good idea. Thanks for suggesting it. I've added the comment in [b5dc87f](https://github.com/Automattic/jetpack/pull/43413/commits/b5dc87f16a2dc962c76289137c49a8c6c1696ea2).